portfolio/src/test/java/com/google/sps/HashTableTest.java

```diff
@@ -100,7 +100,7 @@ public void invalidTag(){
     //Tag should not be supported so should return empty hash table.
     BlogHashMap actual = new BlogHashMap();
     List<BlogMessage> temp = new ArrayList<>();
-    temp.add(new BlogMessage(1, "#Edm", "I listen to Edm while I work.", "", "Steven", new ArrayList<String>(), System.currentTimeMillis()));
+    temp.add(new BlogMessage((long) 1, "#Edm", "I listen to Edm while I work.", "", "Steven", "", new ArrayList<String>(), System.currentTimeMillis()));
```

Comment:
For long constants, you don't need the (long) conversion, just append a "l" character after the number to tell the compiler to use a long integer here.